### ISSUE TITLE
Current HookaimCode offset?

### ISSUE THREAD

**Issue by tronico88** [HKbool/Pubg_d2d_code#1]
cant seem to find the right offset for this. is the sig still working?
`*(ULONG64 *)(HookAimCode + 0x2) = GameBase + 0x42504C;    //48 6B CF 58          48 03 8B 200B0000    74 17  48 8D 55 E7`

**Comment by HKbool** [comment]
You need a new address

**Comment by HKbool** [comment]
@tronico88  You need a new address

**Comment by tronico88** [comment]
yes but can i use that sig u provided? because im not getting any clear results.. or how would i find a new address?

**Comment by CalvinRui** [comment]
替换了最新基址，编译成功后运行EXE没有效果
